src/events/components/CalendarView/index.tsx

```diff
@@ -76,7 +76,13 @@ export const CalendarView = () => {
         <CalendarFillerTiles days={previous} />
         {displayEventMonth &&
           displayEventMonth.map((events, index) => (
-            <CalendarTile key={`${month.toFormat('yyyy-MM')}-${index + 1}`} events={events} day={index + 1} active />
+            <CalendarTile
+              key={`${month.toFormat('yyyy-MM')}-${index + 1}`}
+              events={events}
+              day={index + 1}
+              month={month}
```

Comment:
I think it is quite strange that our CalendarTile component doesn't know which date it is displaying, like as a whole date object. I would blame myself for that.

What if we removed the `day: number` and `month: DateTime` props, and just replace it with a `date: DateTime` prop which is the actual year, month and day the Tile is supposed to represent?

I think something like this would work, but that is without testing it:
``` typescript
// Taking the month object, and setting the day to be the day of the tile
const date = month.set({ day: index + 1 })

// How it could look in the JSX
<CalendarTile
  key={`${month.toFormat('yyyy-MM')}-${index + 1}`}
  events={events}
  date={month.set({ day: index + 1 })}
  active
/>
```

This could also make the `isToday` comparison a lot simpler.
We could possibly just do something like:
``` typescript
  // Get the current date as a Luxon DateTime
  const today = DateTime.local();
  // Compare just the string representation of the year, month and day
  const isToday = date.toFormat('yyyy-MM-dd') === today.toFormat('yyyy-MM-dd');
```

Reply:
This comment is fixed in #360, is it not @oleast?

Reply:
Yes :smile: